packages/app/tailwind.config.js

```diff
@@ -4,6 +4,12 @@ module.exports = {
   darkMode: false, // or 'media' or 'class'
   theme: {
     extend: {
+      letterSpacing: {
```

Comment:
I definitely don't like the idea that we have two themes defined, one in dripsy and one in tailwind. This may lead to inconsistencies. For now, I'd suggest adding this font to dripsy as well, however it's something we need to discuss.